File: 2025/intermediario/2025.1/1036.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    double a, b, c, delta, r1, r2;
    cin >> a >> b >> c;
    delta = pow(b, 2) - 4 * a * c;
    cout << fixed << setprecision(5);
    if (delta > 0 && a > 0){
        r1 = (-b + sqrt(delta)) / (2 * a);
        r2 = (-b - sqrt(delta)) / (2 * a);
        cout << "R1 = " << r1 << "\n";
        cout << "R2 = " << r2 << "\n";
    } else {
        cout << "Impossivel calcular" << "\n";
    }
    return 0;
}

File: 2025/iniciante/2025.1/14-04-2025/1036.py
a, b, c = map(float, input().split())
delta = b**2 - 4 * a * c
if a > 0 and delta > 0:
    r1 = (-b + delta**0.5) / (2 * a)
    r2 = (-b - delta**0.5) / (2 * a)
    print(f"R1 = {r1:.5f}")
    print(f"R2 = {r2:.5f}")
else:
    print("Impossivel calcular")
